### DISCUSSION TITLE
Terminal considered harmful

### DISCUSSION THREAD

**Discussion by e-liouta** [upatras-hci/iv#61] [Show and tell]
![image](https://user-images.githubusercontent.com/17378923/206178372-b370f7b2-9142-4279-a99c-ab4a25a4be59.png)

**Comment by epidrome** [comment]
@e-liouta διασκεδαστικό, αλλά επειδή υπάρχει ο κίνδυνος κάποιος να το πιστέψει ως πραγματικό,

απαντάω με τεκμηρίωση (όχι δικιά μου, από νομπελίστες):

“Simple things should be simple; complex things should be possible.“ Alan Kay

η γραφική διεπαφή στην βασική δημοφιλή μορφή της δεν επιτρέπει τίποτα δύσκολο, η γραμμή εντολών **φαίνεται** δυσκολότερη από την γραφική αλλά δεν είναι: https://www.youtube.com/watch?v=XvDZLjaCJuw

ο λόγος που η γραμμή εντολών φαίνεται δύσκολη σε κάποιον που έμαθε πρώτα την γραφική διεπαφή είναι επίσης τεκμηριωμένος: https://en.wikipedia.org/wiki/Imprinting_(psychology)#Baby_duck_syndrome
